zip-0226.rst

```diff
@@ -302,30 +302,90 @@ Burn Mechanism Consensus Rules
+The transaction format closely follows the version 5 transaction format described in the Zcash specification [#protocol-transactionstructure]_. 
+The Common Transaction Fields, Transparent Transaction Fields, and Sapling Transaction Fields remain the same as in the version 5 transaction format. 
+We make some modifications to the Orchard Transaction Fields and the Orchard Action Descriptions, which we detail in the first and second tables below respectively. 
+We also add ZSA Burn Fields to the transaction format, which we detail in the first table below.
+For brevity, we omit the descriptions from the table below unless they differ from the descriptions in §7.1 of the Zcash specification [#protocol-transactionstructure]_.
+
+
++-----------------------------+--------------------------+-------------------------------------------+---------------------------------------------------------------------+
+| Bytes                       | Name                     | Data Type                                 | Description                                                         |
++=============================+==========================+===========================================+=====================================================================+
+| **Orchard Transaction Fields**                                                                                                                                           |
++-----------------------------+--------------------------+-------------------------------------------+---------------------------------------------------------------------+
+|``varies``                   |``nActionsOrchard``       |``compactSize``                            |                                                                     |
+|                             |                          |                                           |                                                                     |
++-----------------------------+--------------------------+-------------------------------------------+---------------------------------------------------------------------+
+|``852 * nActionsOrchard``    |``vActionsOrchard``       |``ZSAOrchardAction[nActionsOrchard]``      |A sequence of ZSA Orchard Action descriptions, encoded per           |
+|                             |                          |                                           |the `ZSA Orchard Action Description Encoding`_. **[UPDATED FOR ZSA]**|
++-----------------------------+--------------------------+-------------------------------------------+---------------------------------------------------------------------+
+|``1``                        |``flagsOrchard``          |``byte``                                   |                                                                     |
```

Comment:
We will need a spec for `flagsOrchard` and how it is translates to `finilized`. (a different pr)